mlops/parallelm/mlops/mlops.py

```diff
@@ -235,6 +241,14 @@ def done(self):
         self._done_called = True
         self._init_called = False
 
+    @property
+    def get_uuid(self):
```

Comment:
As long as it is a property, maybe call it just 'uuid'

Reply:
ive removed the property decorator - i dont think i have used it the right way - will revise it